packages/graphql/tests/integration/aggregations/where/AND-OR-operations.int.test.ts

```diff
@@ -174,274 +174,4 @@ describe("Nested within AND/OR", () => {
             ],
         });
     });
-
-    test("AND within an AND", async () => {
-        const query = `
-            query {
-                ${postType.plural}(where: { 
-                    likesAggregate: {
-                        AND: [
-                            { count_LTE: 2 }
-                            {
-                                AND: [
-                                    {
-                                        node: {
-                                            testString_SHORTEST_LT: 4
-                                        }
-                                    }
-                                    {
-                                        node: {
-                                            testString_EQUAL: "${testString5}"
```

Comment:
In this case, I would change the operator instead of deleting the whole test. This test assertion is not about this specific filter but about testing the logical operator with aggregation.